src/Hit/Formatting.hs

```diff
@@ -0,0 +1,34 @@
+module Hit.Formatting
+  ( padLeft
+  , padRight
+  , maxLenOn
+  )
+where
+
+import qualified Data.Text                     as T
+
+-- | @padLeft n t@ pads the text 't' with spaces on the left until it reaches length 'n'.
+--
+-- @
+-- padLeft 10 "hello"' ≡ "·····hello"
```

Comment:
Could we use raw spaces in the documentation instead of `·`? Not everybody is familiar with this style of showing whitespaces.